code/lib/preview-api/src/modules/store/csf/prepareStory.ts

```diff
@@ -243,19 +243,6 @@ function preparePartialAnnotations<TRenderer extends Renderer>(
     initialArgsBeforeEnhancers
   );
 
-  // Add some of our metadata into parameters as we used to do this in 6.x and users may be relying on it
-
-  if (!global.FEATURES?.breakingChangesV7) {
```

Comment:
Should this be removed? I wonder if it's needed somehow in case `storyStoreV7` is false

Reply:
Is breakingChangesV7 not always `true` in SB 7.0 (or must be)?

Reply:
@yannbf I don't think it's necessarily to do with ssv7, just an old implementation detail that we used to store these things in parameters. We never told people to read them from there (AFAIK), but we were playing it safe in 6.x. 

I think we can definitely drop this code in 7.0, but we may have missed the boat.

Not sure about `breakingChangesV7` entirely. Certainly it makes sense to have gotten rid of it (I thought we already did, but it looks like @ndelangen just reversed the default). But other places it has an effect like [this](https://github.com/storybookjs/storybook/blob/3b9b267e11eec21fe298c5386ca32be1628f09ae/code/lib/core-server/src/build-static.ts#L162) I am less sure of.